### PR TITLE
Removed all crafting on a stick items

### PR DESCRIPTION
**Pull request by Auseawesome**
Also edited recipe for the stonecutter so it can be made without any covers if you somehow lose it...

This pull request will be merged when its done, when the following checkboxes are checked and when it passes review.
[X] - tested and confirmed working
    [X] - singleplayer
    [X] - server
    [X] - all test cases
        [X] - items properly removed using removeItems()
        [X] - new recipe for stonecutter on a stick works

### CODE REVIEW

**Comment by Auseawesome** [comment]
Don't merge this, just realised it's in the wrong script

**Comment by Auseawesome** [comment]
That should now be fixed

File: kubejs/server_scripts/base/mods/craftingOnAStick.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+if (feature('Remove crafting tables on a stick')) {
+    removeRecipe({mod:'crafting_on_a_stick'})
+    let namespace = (string) => {
```
Comment (Yarden-zamir) [inline kubejs/server_scripts/base/mods/craftingOnAStick.js:3]: I dont like ommiting the modname as it makes it harder to refactor later. Also in this case it doesnt save you work or adds redability so i dont think it helps you anyway. I'd prefer we use the full ids and forgo this function

**Review by Yarden-zamir — CHANGES_REQUESTED** [review]

**Review by Yarden-zamir — APPROVED** [review]

**Comment by Auseawesome** [comment]
Lemme remove the feature.json file quickly, this was before the last pull which fixed everything

**Pull request merged**